api/inventorypb/agents.proto

```diff
@@ -893,6 +917,57 @@ message ChangeRDSExporterResponse {
   RDSExporter rds_exporter = 1;
 }
 
+// Add/Change ExternalExporter
+
+message AddExternalExporterRequest {
+  // The node identifier where this instance is run.
+  string runs_on_node_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Service identifier.
+  string service_id = 2;
+  // HTTP basic auth username for collecting metrics.
+  string username = 3;
+  // HTTP basic auth password for collecting metrics.
+  string password = 4;
+  // Scheme to generate URI to exporter metrics endpoints.
+  bool scheme = 6;
+  // Disable basic metrics.
+  string metric_path = 7 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Listen port for scraping metrics.
+  uint32 listen_port = 8 [
+    (validator.field) = {
+      int_gt: 0
+      int_lt: 65535
```

Comment:
65536

Reply:
I set this based on readme https://github.com/mwitkow/go-proto-validators, but I checked generated code and you are right